.github/ISSUE_TEMPLATE/enhancement.yml

```diff
@@ -0,0 +1,74 @@
+#
+# Copyright (c) 2019 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at:
+#
+#     https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+#
+
+name: Enhancement💡
+description: Suggest a enhancement for Eclipse JKube
+labels: [enhancement]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ### Suggesting a new enhancement💡
+        🔍  Before opening a new issue please search existing issues at https://github.com/eclipse/jkube/issues
+
+        🤔  To make it easier for us to help you, please include as much useful information as possible.
+
+        Useful Links:
+
+          - 📄  Documentation: https://www.eclipse.org/jkube/docs/
+          - 📝  Contributing: https://www.eclipse.org/jkube/contributing
+
+
+  - type: dropdown
+    id: component 
+    attributes:
+      label: Component
+      description: Component
+      options:
+        - "Kubernetes Maven Plugin"
+        - "Kubernetes Gradle Plugin"
+        - "OpenShift Maven Plugin"
+        - "OpenShift Gradle Plugin"
+        - "JKube Kit"
```

Comment:
Might be confusing for newer users who don't understand the underlying architecture. Either make it non-required, or provide an `other` option.

e.g. I want you to provide an annotation processor. I want you to provide support for Bazel. And so on.